Post/tests.py

```diff
@@ -1,13 +1,45 @@
 from Post.models import Post
+from django.db.models.query import QuerySet
+import pytest
 
 
-class TestPost():
+@pytest.mark.django_db
+def test_post_str(get_post):
```

Comment:
@Shoval-Ovadia Please change 'get_post' to 'post' it will be better and more like the convention as I wrote down here